engine/graphics/src/webgpu/graphics_webgpu_private.h

```diff
@@ -24,218 +24,218 @@
 #include "../graphics_private.h"
 
 #ifdef __EMSCRIPTEN__
-# include <webgpu/webgpu.h>
+#include <webgpu/webgpu.h>
 #endif
 
 namespace dmGraphics
 {
-    const static uint32_t MAX_TEXTURE_COUNT  = 32;
-    const static uint8_t MAX_VERTEX_BUFFERS = 2;
+    const static uint32_t MAX_TEXTURE_COUNT = 32u;
+    const static uint8_t MAX_VERTEX_BUFFERS = 2u;
 
     struct WebGPUBuffer
     {
-        WebGPUBuffer(WGPUBufferUsageFlags usage) : m_Usage(usage) { }
+        WebGPUBuffer(WGPUBufferUsageFlags usage)
+            : m_Usage(usage)
+        {
+        }
 
-        WGPUBuffer                      m_Buffer = NULL;
-        const WGPUBufferUsageFlags      m_Usage;
+        WGPUBuffer m_Buffer = NULL;
+        const WGPUBufferUsageFlags m_Usage; // uint32_t
 
-        size_t                          m_Size = 0;
-        size_t                          m_Used = 0;
+        size_t m_Size = 0;
+        size_t m_Used = 0;
```

Comment:
I don't think you should change the layout/style here regarding aligned members. This was modeled after the other adapter headers and it is a very convenient style for getting an overview of the struct.

Reply:
Fixed here and reformated
https://github.com/defold/defold/pull/9465/files#diff-1026e0038b722990204a42bed8a6f7c0ec2302aa79e3fad1959d62ba968edfa2R6